README.md

```diff
@@ -162,6 +161,26 @@ $ ./cts create -n4 unassigned drafting # create 4 cases each in unassigned and d
 $ ./cts --help create                  # display full help text for create command
 ```
 
+### Dumping the database
+
+We have functionality to create an anonymised copy of the database on production or staging. This feature is to be used as a very last resort. If the copy of the database is needed for debugging please consider the following options first:
+- seeing if the issue is covered in the feature tests
+- trying to track the issue through Kibana
+- recreating the issue locally
+
+If the options above do not solve the issue you by create an anonymised dump of the database by running
```

Comment:
Looking good! How about mentioning the task `rake db:dump:help` for further instructions on how to configure the dump?